Library/Homebrew/dev-cmd/dispatch-build-bottle.rb

```diff
@@ -27,8 +27,12 @@ def dispatch_build_bottle_args
              description: "Upload built bottles."
       switch "--linux",
              description: "Dispatch bottle for Linux (using GitHub runners)."
+      switch "--linux-self-hosted",
+             description: "Dispatch bottle for Linux (using self-hosted runner)."
+      switch "--wheezy",
+             description: "Use Debian Wheezy container for building the bottle on Linux."
```

Comment:
How about `--linux-wheezy` here?

Reply:
Or even `linux-debian-wheezy`?

Reply:
Isn't the description enough :)?

Reply:
Feels a bit weird to me not not reference it in this flag but have it as `--linux-*` in others.